tests/openvino/native/quantization/test_sanity.py

```diff
@@ -27,7 +27,7 @@
 
 OMZ_MODELS = [
     ('resnet-18-pytorch', 'imagenette2-320', {'accuracy@top1': 0.784, 'accuracy@top5': 0.946}),
-    ('mobilenet-v3-small-1.0-224-tf', 'imagenette2-320', {'accuracy@top1': 0.733, 'accuracy@top5': 0.913}),
+    ('mobilenet-v3-small-1.0-224-tf', 'imagenette2-320', {'accuracy@top1': 0.74, 'accuracy@top5': 0.913}),
```

Comment:
@alexsu52 I think that changes in this PR also may improve accuracy for other models.

Reply:
Need to check it. You can use conformance post training tests.

Reply:
Reverted